scripts/generateNTPSponsoredImages.js

```diff
@@ -9,18 +9,27 @@ const request = require('request')
 const commander = require('commander')
 const admZip = require('adm-zip')
 
+const getRegionList = () => {
+  return [ 'en-US' ];
```

Comment:
Nit: I think you don't need the semicolon here, if this is meant to follow the standard style. Might want to run it through the linter if you haven't already.

Reply:
Oh, I did it unconsciously :)